### PR TITLE
Add descriptions to children docs-Data Hub Guide and the rest guides

### PR DESCRIPTION
**Pull request by Yiyun333**

### CODE REVIEW

File: content/en/docs/addons/apd-addon/ig/_index.md [hunk 1]
```diff
@@ -2,6 +2,7 @@
 title: "APD Installation Guides"
 url: /addons/apd-addon/ig/
 category: "APD"
+description: "Presents a list of APD/APM installation guides according to the APD/APM version."
```
Comment (MarkvanMents) [inline content/en/docs/addons/apd-addon/ig/_index.md]: "according to the" sounds a little formal to me. 
You could change it to something like "for each".
But that is just a personal thing - it is also OK as it is.

**Review by MarkvanMents — CHANGES_REQUESTED** [review]
Great,
I've made one comment on what you have done, but it is a personal preference rather than anything wrong.

There are two changes in files which aren't included in this PR:
* Missing a description for `/apidocs-mxsdk/mxsdk/_index.md` which appears in [APIs and SDK](https://docs.mendix.com/apidocs-mxsdk/). It also has a description list, but no descriptions. Perhaps change `description_list: true` to `simple_list:true`? Or perhaps this is in a different PR?
* There is also a typo in the description of /data-hub/share-data/_index.md - "and create a new apps that consumes this asset." should be "create a new **app**" (not your typo - but could you change it?)

If you can address those two docs, then go ahead and publish. You don't need further approval.

**Comment by Yiyun333** [comment]
> * Missing a description for `/apidocs-mxsdk/mxsdk/_index.md` which appears in [APIs and SDK](https://docs.mendix.com/apidocs-mxsdk/). It also has a description list, but no descriptions. Perhaps change `description_list: true` to `simple_list:true`? Or perhaps this is in a different PR?

Hi Mark, Luyao told me that there will be some restructuring for the "Mendix Platform SDK" category and some docs will be relocated there. She told me that after she is done restructuring, she will adjust the descriptions for this catogory but I will also send your comment to her.

**Comment by Yiyun333** [comment]
> * There is also a typo in the description of /data-hub/share-data/_index.md - "and create a new apps that consumes this asset." should be "create a new **app**" (not your typo - but could you change it?)

Thanks for spotting this! I've changed it.

File: content/en/docs/addons/apd-addon/ig/_index.md (hunk 1 above)
Comment (Yiyun333) [reply content/en/docs/addons/apd-addon/ig/_index.md]: Thanks for the suggestion. I also think "for each" is nicer here. I've made the changes.

**Comment by Yiyun333** [comment]
> > * Missing a description for `/apidocs-mxsdk/mxsdk/_index.md` which appears in [APIs and SDK](https://docs.mendix.com/apidocs-mxsdk/). It also has a description list, but no descriptions. Perhaps change `description_list: true` to `simple_list:true`? Or perhaps this is in a different PR?
> 
> Hi Mark, Luyao told me that there will be some restructuring for the "Mendix Platform SDK" category and some docs will be relocated there. She told me that after she is done restructuring, she will adjust the descriptions for this catogory but I will also send your comment to her.

@Luyao-Zhang-1 and I just discussed your comment. She will address this issue after the restructuring is done: [TW1848](https://mendix.atlassian.net/browse/TW-1848).

**Review by MarkvanMents — APPROVED** [review]

**Pull request merged**